src/simcore_service_deployment_agent/exceptions.py

```diff
@@ -15,6 +15,13 @@ def __init__(self):
         super().__init__(msg)
 
 
+class TagSyncErrorException(AutoDeployAgentException):
+    """Tag sync activated but latest tags dont match"""
+
+    def __init__(self, msg):
```

Comment:
MINOR: there is also room for improvement in the exceptions. This is not only verbose but not very effective way to use exceptions.

Reply:
we can have a look, these were inherited and I dont have the knowhow what is considered good/bad practice w.r.t. writing your own exceptions, thx